recipes/mcmc-sampling-emcee/mcmc_sampling.ipynb

```diff
@@ -45,7 +45,7 @@
   },
```

Comment:
<p>I would rather make this a markdown cell, to make it more clear.</p><p>Then for the code you could utilise the <code>python</code> code functionality of markdown? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy-recipes/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>